src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/PlaygroundResultFeatureStrategyList/PlaygroundResultStrategyList/PlaygroundResultFeatureStrategyItem/PlaygroundResultStrategyExecution/PlaygroundResultStrategyExecutionParameters/PlaygroundResultStrategyExecutionParameters.tsx

```diff
@@ -0,0 +1,136 @@
+                                    percentage={percentage}
+                                    size="2rem"
+                                />
+                            </Box>
+                            <div>
+                                <Chip
+                                    color="success"
+                                    variant="outlined"
+                                    size="small"
+                                    label={`${percentage}%`}
+                                />{' '}
+                                of your base{' '}
+                                {constraints.length > 0
+                                    ? 'who match constraints'
+                                    : ''}{' '}
+                                is included.
+                            </div>
+                        </Box>
+                    );
+                case 'userIds':
+                case 'UserIds':
+                    const users = parseParameterStrings(parameters[key]);
+                    return (
+                        <PlaygroundConstraintItem
+                            key={key}
+                            value={users}
+                            text="user"
+                            input={
+                                Boolean(input?.context?.[getMappedParam(key)])
+                                    ? input?.context?.[getMappedParam(key)]
+                                    : 'no value'
+                            }
+                            showReason={
+                                Boolean(input?.context?.[getMappedParam(key)])
+                                    ? !users.includes(
+                                          input?.context?.[getMappedParam(key)]
+                                      )
+                                    : undefined
+                            }
+                        />
+                    );
+                case 'hostNames':
+                case 'HostNames':
+                    const hosts = parseParameterStrings(parameters[key]);
+                    return (
+                        <PlaygroundConstraintItem
+                            key={key}
+                            value={hosts}
+                            text={'host'}
+                            input={
+                                Boolean(input?.context?.[getMappedParam(key)])
+                                    ? input?.context?.[getMappedParam(key)]
+                                    : 'no value'
+                            }
+                            showReason={
+                                Boolean(input?.context?.[getMappedParam(key)])
+                                    ? !hosts.includes(
+                                          input?.context?.[getMappedParam(key)]
+                                      )
+                                    : undefined
+                            }
+                        />
+                    );
```

Comment:
This might be changed with https://github.com/Unleash/unleash/pull/1889. Don't know if we want to integrate that or not 💁🏼